docs/source/eventing/actions/mqtt_publish.rst

```diff
@@ -0,0 +1,42 @@
+.. |yes| image:: ../../images/yes.png
+.. |no| image:: ../../images/no.png
+
+.. role:: underline
+   :class: underline
+
+Mqtt_publish
```

Comment:
Shouldn't this be `MQTT publish`?

Reply:
"mqtt_publish" is the real name of the action